lib/libs/bitcoin_lib/lib/src/protocol/types/script.dart

```diff
@@ -0,0 +1,186 @@
+import 'dart:typed_data';
+
+import 'package:meta/meta.dart';
+
+import '../../extensions/int_extensions.dart';
+import 'op_code.dart';
+import 'script_pubkey.dart';
+import 'script_sig.dart';
+import 'script_type.dart';
+import 'variable_length_integer.dart';
+
+@immutable
+class Script {
+  const Script(this.length, this.commands, this.type);
+
+  // ignore: prefer_constructors_over_static_methods
+  static Script deserialize(Uint8List bytes) {
```

Comment:
I added this comments to ignore prefer_constructors_over_static_methods in this method to call from factory constructor in ScriptPubKey and ScriptSig.